qds_sdk/clusterv2.py

```diff
@@ -321,11 +341,18 @@ def set_data_disk(self,
         self.cluster_info['datadisk']['encryption'] = enable_encryption
 
     @staticmethod
+    def list_info_parser(argparser, action):
+        argparser.add_argument("--state", dest="label",
+                               help="State of the cluster")
```

Comment:
List parser in `cluster.py` supports passing following arguments
```
group.add_argument("--id", dest="cluster_id",
                           help="show cluster with this id")
group.add_argument("--label", dest="label",
                           help="show cluster with this label")
group.add_argument("--state", dest="state", action="store",
                           choices=['up', 'down', 'pending', 'terminating'],
                           help="list only clusters in the given state")

```

I think the same should be supported here as well.